lua/terrortown/lang/zh_hans.lua

```diff
@@ -1889,66 +1887,66 @@ L.slot_weapon_extra = "额外槽"
 L.slot_weapon_class = "职业槽"
 
 -- 2023-10-04
---L.label_voice_duck_spectator = "Duck spectator voices"
---L.label_voice_duck_spectator_amount = "Spectator voice duck amount"
---L.label_voice_scaling = "Voice Volume Scaling Mode"
---L.label_voice_scaling_mode_linear = "Linear"
---L.label_voice_scaling_mode_power4 = "Power 4"
---L.label_voice_scaling_mode_log = "Logarithmic"
+L.label_voice_duck_spectator = "降低观察者语音"
```

Comment:
```suggestion
L.label_voice_duck_spectator = "观察者语音淡化"
```

Closest thing to a localization to the technical term "audio ducking" seems to be "音频闪避", which literally reads "audio evasion". This seems very obscure and unintuitive, so I elected to use "淡化" (fade out) instead.